lib/delta-file.js

```diff
@@ -67,7 +63,14 @@ export default class DeltaFile {
   async load() {
     try {
       await this.download();
-      const changeSets = await fs.readJson(this.filePath, { encoding: 'utf-8' });
+      let fileStream = fs.createReadStream(this.filePath);
```

Comment:
I'm always a bit confused about streams and their events in an async method. I assume that the final `await json` will throw any pipeline errors as they cascade to the final consumer in the stream pipeline. If that's the case this looks fine.